CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ### Unreleased
+  - Fixed inverted constants in survey transform.
```

Comment:
Changelog messages should be in the imperative (I think that's the right word) tense - so they'd consistently start with 'Add/Remove/Update/Fix' etc rather than past tense. Admittedly this isn't documented, so I'll raise a ticket now for documenting this.